### ISSUE TITLE
Wallet design - Transaction history/Pending transactions

### ISSUE THREAD

**Issue by sla-shi** [status-im/status-react#1524]
### User Story

As a user, I want to access transaction history so that I can review completed and pending transactions. 

#### Transactions History

- [x] I can access the Transactions history from the Wallet main screen and go back 
- [x] I can see the history of transactions and distinguish completed and pending ones
- [x] I can distinguish transactions in the list by transaction type using color icons: inbound, outbound, pending and postponed
- [x] I can see the list of Unsigned transactions (empty for now)
- [x] I can refresh transactions history by pull-to-refresh gesture
- [x] I can see the error message when something goes wrong

Outbound flows should be presented but not required to be implemented in the scope if this issue (each flow will have its own acceptance):
- [x] I can sign the unsigned transaction
- [x] I can delete the unsigned transaction
- [ ] I can filter transactions

**Comment by sla-shi** [comment]
@sla-shi to create the separate binding task.

**Comment by denis-sharypin** [comment]
@jeluard All UI screens are here zpl://screen?pid=5863cdcee68455850f1929f2&sid=5991fb22d214630c91087f0f

**Comment by asemiankevich** [comment]
<s>https://github.com/status-im/status-react/issues/1692 is blocking to finish testing</s> . Tabs are not implemented yet

**Comment by jeluard** [comment]
Missing:

* specs for data format
* usage of real wallet info
* real-data for history and unsigned transactions
* filter binding
* filter persistency
* tabs are currently broken (needs some refactring to be reusable)
* error handling

**Comment by oskarth** [comment]
https://github.com/status-im/status-react/pull/1783 has been merged now.

Moving to blocked as there's nothing else to review, and it is blocked by other sub-issues. @jeluard @sla-shi if you think it should be "parked" somewhere else please move it where you think it makes sense. It shouldn't be in "ready for review/test" until there's some action reviewers and/or QA can take though.

**Comment by annadanchenko** [comment]
#1801 needs to be fixed

**Comment by asemiankevich** [comment]
https://github.com/status-im/status-react/issues/1801 is fixed and closed

List of issues to be fixed:
https://github.com/status-im/status-react/issues/1824
https://github.com/status-im/status-react/issues/1814
https://github.com/status-im/status-react/issues/1816
https://github.com/status-im/status-react/issues/1813
https://github.com/status-im/status-react/issues/1829

**Comment by asemiankevich** [comment]
https://github.com/status-im/status-react/issues/1854 is logged as regression result

**Comment by sla-shi** [comment]
#1824 fixed
#1813 fixed
#1829 fixed

#1814 open, not critical
#1816 open, not critical

**Comment by jeluard** [comment]
All linked issues are fixed.

[Transactions filtering](https://github.com/status-im/status-react/issues/2228) has been pushed to a later release.
